pyproject.toml

```diff
@@ -5,21 +5,31 @@ description = ""
 authors = ["Christopher Navarro <cmnavarr@illinois.edu>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
-fastapi = "^0.73.0"
-uvicorn = "^0.17.6"
-gunicorn = "^20.1"
-alembic = "^1.7.7"
-psycopg2-binary = "^2.9.3"
-SQLAlchemy = {extras = ["mypy"], version = "~1.4"}
-passlib = {extras = ["bcrypt"], version = "~1.7"}
-pydantic = "^1.9.0"
-python-dotenv = ">=0.10.3"
+python = ">=3.10"
```

Comment:
You will need to update the Dockerfile to 3.10 since it currently using the 3.8 image

Reply:
@Rashmil-1999 A quick fix so this branch works would be to change Dockerfile to FROM python:3.10-slim so that the branch works and can be deployed.